app/src/main/java/uk/ac/ebi/atlas/search/JsonGeneSearchController.java

```diff
@@ -236,7 +238,7 @@ public Boolean isMarkerGene(@RequestParam MultiValueMap<String, String> requestP
                                 .map(Map.Entry::getKey)
                                 .collect(toImmutableSet()));
 
-        return markerGeneFacets != null && markerGeneFacets.size() > 0;
+        return markerGeneFacets != null && !markerGeneFacets.isEmpty();
```

Comment:
We can use `StringUtils.isBlank()`-> this would check white spaces and null.

Reply:
@upendrakumbham `markerGeneFacets` is not a `String`. Its type is: `ImmutableMap<String, Map<String, Map<Integer, List<Integer>>>>`. Could you tell me please where would you like to use `StringUtils.isBlank()`?
What I am checking here is: are there any marker gene based on the given request parameters.